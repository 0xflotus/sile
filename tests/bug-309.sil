\begin[papersize=a6]{document}
\nofolios
\set[parameter=document.parskip,value=2em plus 1em]
Test par

Test par
\end{document}
