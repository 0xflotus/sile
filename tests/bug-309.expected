<tests/bug-309.sil>
Set paper size 	297.6377985	419.5275636
Begin page
Mx 	34.8819
My 	27.2264
Set font 	Gentium;10;400;;normal;;LTR
T	52 69 83 84	(Test)
Mx 	54.9797
T	80 65 82	(par)
Mx 	34.8819
My 	59.2264
T	52 69 83 84	(Test)
Mx 	54.9797
T	80 65 82	(par)
[1] End page
Finish

